storage/volume/driver/driver.go

```diff
@@ -82,44 +87,88 @@ func (m VolumeStoreMode) UseLocalMeta() bool {
 	return (m & UseLocalMetaStore) != 0
 }
 
-//
-type driverTable map[string]Driver
-
-// Add is used to add driver into driver table.
-func (t driverTable) Add(name string, d Driver) {
-	t[name] = d
+// driverTable contains all volume drivers
+type driverTable struct {
```

Comment:
You could use `type SafeMap struct` in package `pkg/collect`, instead of declaring a brand new one.

Reply:
Now driverTable only has a field named drivers except sync.Mutex. In future we may add more fields, like `alias drivers` and so on. And the SafeMap may not safisfy my requirements. For example we want add a driver, first we will check whether the driver existed. If not, we add the driver. This action is atomic. Of course, we can change the SafeMap

Reply:
> In future we may add more fields

You can define like this, I think the following :
```
type driverTable struct {
    sync.Mutex
    drivers collect.SafeMap
    alias xxxType
}
```

Yes, of course. We could enhance SafeMap. 😄